helm_deploy/create-and-vary-a-licence/values.yaml

```diff
@@ -88,5 +88,45 @@ gotenberg:
     runAsUser: 1001
     runAsNonRoot: true
 
+delius-wiremock:
+  nameOverride: delius-wiremock
+  replicaCount: 2
+
+  image:
+    repository: smcveigh941/delius-wiremock
+    tag: latest
+    port: 5000
+
+  namespace_secrets:
+    create-and-vary-a-licence:
+      SYSTEM_CLIENT_ID: "SYSTEM_CLIENT_ID"
+      SYSTEM_CLIENT_SECRET: "SYSTEM_CLIENT_SECRET"
```

Comment:
I assume these string literals point to where the secret it stored, and shouldnt be something like `${SYSTEM_CLIENT_ID}`